src/main/native-package/m4/custom.m4

```diff
@@ -1,8 +1,8 @@
 
 AC_DEFUN([CUSTOM_M4_SETUP],
 [
-  YARA_INC="-I$PWD/../../../libyara/include"
-  YARA_LIB="$PWD/../../../libyara/.libs/libyara.a"
+  YARA_INC="-I$YARA_HOME/libyara/include"
+  YARA_LIB="$YARA_HOME/libyara/.libs/libyara.a"
 
   dnl Update the compiler/linker flags to add yara to the build path
   CFLAGS="$CFLAGS $YARA_INC"
```

Comment:
Same comment as above - let's default to the current behavior ($PWD/../../../) is YARA_HOME env var is not set